src/components/settings/About.vue

```diff
@@ -10,7 +10,9 @@
                 ⚠️ {{ $t("Frontend Version do not match backend version!") }}
             </div>
 
-            <div class="my-3 update-link"><a href="https://github.com/louislam/uptime-kuma/releases" target="_blank" rel="noopener">{{ $t("Check Update On GitHub") }}</a></div>
+            <div class="my-2 update-link"><a href="https://github.com/louislam/uptime-kuma/releases" target="_blank" rel="noopener">{{ $t("Check Update On GitHub") }}</a></div>
+
+            <div class="my-2 update-link">Country Flag Emoji Polyfill by <a href="https://github.com/twitter/twemoji" target="_blank" rel="noopener">@Twemoji</a></div>
```

Comment:
yea.. adding this in the middle of the `Check Update On GitHub` and `Show update if available` button is really not what I meant.

Please move it all the way to the bottom and add a new section (with a representative title, like in other software)

Also please have a look at how we translate strings. Currently, our awsome translators could not translate this.